lib/package/plugins/policyMetaData.json

```diff
@@ -48,7 +48,7 @@
   "ResponseCache" :{"indications":["responsecache","rc","cache","^responsecache$"]},
   "Script" :{"indications":["script","scr"]},
   "ServiceCallout" :{"indications":["callout","sc"]},
-  "SetDialogResponse" :{"indications":["setdialogresponse","dialogflow","sdr","^setdialogresponse$"], "profiles":["apigeex"] },
+  "SetDialogflowResponse" :{"indications":["setdialogresponse","dialogflow","sdr","^setdialogresponse$"], "profiles":["apigeex"] },
```

Comment:
`{"indications":["setdialogresponse","dialogflow","sdr","^setdialogresponse$"], "profiles":["apigeex"] }`

should we rather do
`{"indications":["setdialogflowresponse","dialogflow","sdr","^setdialogflowresponse$"], "profiles":["apigeex"] }`

Reply:
done